packages/cli/src/utils.ts

```diff
@@ -336,6 +336,9 @@ export function dtsTemplate(
 }
 
 function getReexportStatement(namedExports: string[], source: string): string {
+  if (!namedExports.length) {
+    return "";
```

Comment:
```suggestion
    return `import ${source};`
```

An entrypoint could (and likely does if it has no exports) have side-effects so we still need to import it.

Reply:
Do u always have to be right? :P pushed out this change with extra tests for this